azavea/pfb-network-connectivity
# Add endpoint to serve crash data (#917) and facilitate crash data imports (#918)

## Overview

I've grouped these two issues together since the work in #918 makes it possible to test #917

### Notes

This endpoint still excludes the boundary buffer.

## Testing Instructions

 * Add an unzipped copy of [crashes.zip](https://github.com/azavea/pfb-network-connectivity/files/10127340/crashes.zip) to the data directory in your S3 bucket
 * Inside vagrant, run `scripts/django-manage migrate`
 * Inside vagrant, run `scripts/django-manage import_crash_data`
 * See if you find any crashes at `http://localhost:9301/api/crashes/?uuid=some_analysis_job_uuid`

## Checklist


Resolves #917 
Resolves #918
